codex-rs/core/src/exec.rs

```diff
@@ -76,6 +80,7 @@ pub async fn process_exec_tool_call(
     sandbox_type: SandboxType,
     writable_roots: &[PathBuf],
     ctrl_c: Arc<Notify>,
+    sandbox_policy: SandboxPolicy,
```

Comment:
For a follow-up, but I think we're getting into "`ExecToolCall` should be a struct" territory...